.misc/deps.sh

```diff
@@ -9,12 +9,20 @@ case $CIRCLE_NODE_INDEX in
 esac
 
 # apt-get commands
+sudo add-apt-repository -y ppa:staticfloat/juliareleases
+sudo add-apt-repository -y ppa:staticfloat/julia-deps
 sudo apt-get update
 deps="indent libclang1-3.4 r-base"
 deps_python_dbus="libdbus-glib-1-dev libdbus-1-dev"
 deps_python_gi="glib2.0-dev gobject-introspection libgirepository1.0-dev python3-cairo-dev"
 deps_ruby_npm="gem nodejs"
-sudo apt-get install $deps $deps_python_gi $deps_python_dbus $deps_ruby_npm
+deps_julia="julia"
+deps_julia_packages="Lint"
```

Comment:
I think even deps_julia_packages belongs there, so you don't need to look around the file to find even those^^